nvflare/fuel/hci/server/hci.py

```diff
@@ -68,8 +83,13 @@ def handle(self):
 
             if not conn.ended:
                 conn.close()
-        except BaseException:
-            traceback.print_exc()
+        except BaseException as exc:
+            self.logger.error(f"Admin connection terminated due to exception: {str(exc)}")
+            if self.logger.getEffectiveLevel() <= logging.DEBUG:
+                self.logger.exception("Admin connection error")
```

Comment:
why do we need these two lines?

All the levels below error would already have the "self.logger.error" printed out.

Reply:
The purpose is to only print traceback in debug level so it doesn't clog up prod logs.